relayer/go.mod

```diff
@@ -37,3 +37,5 @@ require (
 	gopkg.in/ini.v1 v1.62.0 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 )
+
+replace github.com/snowfork/go-substrate-rpc-client/v3 => /home/alistair/go-substrate-rpc-client
```

Comment:
Will remove this once related PR is merged. https://github.com/Snowfork/go-substrate-rpc-client/pull/20